### DISCUSSION TITLE
wire:loading.delay.flex at the same time

### DISCUSSION THREAD

**Discussion by lao9s** [livewire/livewire#2523] [1. Help]
How can i use wire:loading delay and flex at the same time?

I tried:
` <x-forms.loading wire:loading.flex wire:loading.delay />`
But no working.

**Comment by FR6** [comment]
It also does not work by combining the modifiers:
```
<div wire:loading.flex.delay>
```

A quick hack to make it work:
```
<div class="flex">
    <div wire:loading.delay>...</div>
</div
```

Livewire version: 2.3

**Comment by howdu** [comment]
Duplicate please see: https://github.com/livewire/livewire/discussions/3004#discussioncomment-1739953

```html
<div wire:loading.block wire:loading.delay>Loading</div>
```
